src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -1334,3 +1334,306 @@ def test_WINNF_FT_S_IPR_7(self, config_filename):
+    device_a['installationParam']['longitude'] = -69.700
+    device_b = json_load(
+        os.path.join('testcases', 'testdata', 'device_b.json'))
+    device_b['installationParam']['latitude'] = 43.902
+    device_b['installationParam']['longitude'] = -69.850
+
+    # Pre-load conditionals and remove reg conditional fields from registration
+    # request.
+    conditional_keys = [
+        'cbsdCategory', 'fccId', 'cbsdSerialNumber', 'airInterface',
+        'installationParam', 'measCapability'
+    ]
+    reg_conditional_keys = [
+        'cbsdCategory', 'airInterface', 'installationParam', 'measCapability'
+    ]
+    conditionals_b = {key: device_b[key] for key in conditional_keys}
+    device_b = {
+        key: device_b[key]
+        for key in device_b
+        if key not in reg_conditional_keys
+    }
+
+    # Load grant requests.
+    grant_a = json_load(
+        os.path.join('testcases', 'testdata', 'grant_0.json'))
+    grant_b = json_load(
+        os.path.join('testcases', 'testdata', 'grant_0.json'))
+
+    sas_test_harness_config = {
+        'sasTestHarnessName':
+            'SAS-Test-Harness-1',
+        'hostName':
+            getFqdnLocalhost(),
+        'port':
+            getUnusedPort(),
+        'serverCert':
+            getCertFilename('sas.cert'),
+        'serverKey':
+            getCertFilename('sas.key'),
+        'caCert':
+            getCertFilename('ca.cert'),
+        'fullActivityDumpRecords': [
+            generateCbsdRecords([device_a], [[grant_a]])
+        ]
+    }
+
+    domain_proxy = {
+        'registrationRequests': [device_b],
+        'grantRequests': [grant_b],
+        'conditionalRegistrationData': [conditionals_b],
+        'cert': getCertFilename('domain_proxy.cert'),
+        'key': getCertFilename('domain_proxy.key')
+    }
+
+    # Included only as an example; not used below.
+    esc_dpa = {
+        'dpaId': 'East4',
+        # This is the frequency range which will be activated (if applicable)
+        # and checked.
+        'frequencyRange': {
+            'lowFrequency': 3550000000,
+            'highFrequency': 3560000000
+        },
+        'points_builder': 'default (25, 10, 10, 10)',
+        'movelistMargin': 10
+    }
+
+    frequency_range = grant_a['operationParam']['operationFrequencyRange']
+    portal_dpa = {
+        'dpaId': 'BATH',
+        # This is the frequency range which will be checked. One loaded into the
+        # SAS, the DPA is automatically activated.
+        'frequencyRange': {
+            'lowFrequency': frequency_range['lowFrequency'],
+            'highFrequency': frequency_range['highFrequency'],
+        },
+        'points_builder':
+            'default (25, 10, 10, 10)',  # Not actually used since this is a single-point DPA.
+        'movelistMargin': 10
+    }
+
+    config = {
+        'dpaDatabaseConfig': dpa_database_config,
+        'sasTestHarnessConfigs': [sas_test_harness_config],
```

Comment:
Remove peer SAS. It may be easier to assert `sasTestHarnessConfigs` is an empty list at the beginning of test, otherwise you'll need to remove all peer SAS code taken from IPR.7

Reply:
I removed the peer SAS code entirely.